deployer/deployer.yaml

```diff
@@ -173,130 +191,127 @@ items:
       volumes:
       - name: empty
         emptyDir: {}
-      - name: secret
-        secret:
-          secretName: logging-deployer
   parameters:
   -
-    description: "If true, set up to use a second ES cluster for ops logs."
+    description: "The mode that the deployer runs in."
+    name: MODE
+    value: "install"
+  -
+    description: 'Specify prefix for logging components; e.g. for "openshift/origin-logging-deployer:v1.1", set prefix "openshift/origin-"'
+    name: IMAGE_PREFIX
```

Comment:
Because they are needed by the template to even determine what version of
the deployer to run. The template can't read a configmap... well, not yet.
Also, version is one of the things that's likely to vary per invocation.

On Wed, Jun 1, 2016 at 6:10 PM, Eric Wolinetz <notifications@github.com>
wrote:

> In deployer/deployer.yaml
> <https://github.com/openshift/origin-aggregated-logging/pull/158#discussion_r65452473>
> :
>
> >    parameters:
> >    -
> > -    description: "If true, set up to use a second ES cluster for ops logs."
> > +    description: "The mode that the deployer runs in."
> > +    name: MODE
> > +    value: "install"
> > +  -
> > +    description: 'Specify prefix for logging components; e.g. for "openshift/origin-logging-deployer:v1.1", set prefix "openshift/origin-"'
> > +    name: IMAGE_PREFIX
>
> Why aren't we also deprecating the prefix and version?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/158/files/ba393817d375dee51dbea20575a55c9140f9b89e#r65452473>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABz-jUjhYKDoesPTa2lksIhlZb59Jq5ks5qHgNKgaJpZM4Ir7Nu>
> .
>